js/jsmodules.go

```diff
@@ -42,10 +42,8 @@ func getInternalJSModules() map[string]interface{} {
 		"k6/experimental/streams":    streams.New(),
 		"k6/experimental/webcrypto":  webcrypto.New(),
 		"k6/experimental/websockets": &expws.RootModule{},
-		"k6/experimental/timers": newWarnExperimentalModule(timers.New(),
-			"Please update your imports to use k6/timers instead of k6/experimental/timers,"+
-				" which will be removed after September 23rd, 2024(v0.54.0). Ensure your scripts are migrated by then."+
-				" There are no API changes, so this is a drop in replacement."),
+		"k6/experimental/timers": newRemovedModule(
+			"k6/experimental/timers has been graduated, please use k6/timers instead."),
```

Comment:
Aren't timers globally available? :thinking:

Reply:
Hmm ... yeah I guess we can just recommend people to use that instead :thinking:

Reply:
To be honest I mostly opened this so we don't forget to remove them again :)

Reply:
Yep, I got that. I should write this in https://github.com/grafana/k6/pull/3837#discussion_r1674124914 earlier, sorry for that :sweat_smile: